pom.xml

```diff
@@ -13,6 +13,7 @@
 
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
+  <description>Allows to manage slaves running on Unix machines over SSH</description>
```

Comment:
I have changed the text to use launch instead of manage and also agents instead of slaves.